src/lib/services/instance-stats-service.ts

```diff
@@ -111,7 +119,37 @@ export class InstanceStatsService {
         this.logger = getLogger('services/stats-service.js');
     }
 
-    async getToggleCount(): Promise<number> {
+    async start(): Promise<void> {
+        if (!this.snapshotRefresher) {
+            this.refreshStatsSnapshot();
+            this.snapshotRefresher = setInterval(
+                () => this.refreshStatsSnapshot(),
+                minutesToMilliseconds(5),
+            ).unref();
+        }
+    }
+
+    async refreshStatsSnapshot(): Promise<void> {
+        try {
+            this.snapshot = await this.getStats();
+            let appCountReplacement = {};
```

Comment:
Minor detail, this doesn't look like it ever gets reassigned, it can be const
```suggestion
            const appCountReplacement = {};
```